diagnostics-app/src/app/shared/header/header.component.ts

```diff
@@ -21,8 +23,12 @@ export class HeaderComponent {
   @Input() showToggleSideNavButton = true; // Default to showing the toggle button
   @Output() toggleDrawer = new EventEmitter<void>();
   readonly appName = APP_NAME;
+  isSupportAssistMode = this.router.url.startsWith('/support-assist');
```

Comment:
Seems this value will never be changed?
Should we make this a function `isSupportAssistMode()` , or change the value in `onToggleModeChange`?

Reply:
Done. I've changed it to a function getter.